api/clients/node_client.go

```diff
@@ -89,6 +89,7 @@ func (c client) GetChunks(
 	conn, err := grpc.Dial(
 		core.OperatorSocket(opInfo.Socket).GetRetrievalSocket(),
 		grpc.WithTransportCredentials(insecure.NewCredentials()),
+		grpc.WithDefaultCallOptions(grpc.MaxCallRecvMsgSize(32*1024*1024)), // 32 MiB buffer for the worst case
```

Comment:
After erasure coding, if the operator has more than 25% stake, it may exceed 32MiB for 32MiB blob

Reply:
yeah, but I don't intend the buffer to be too big, maybe we should just use a flag with some reasonable default number, what do you think